packages/exchange/migrations/initial.sql

```diff
@@ -0,0 +1,112 @@
+CREATE SCHEMA IF NOT EXISTS public;
```

Comment:
If this is autogenerated and should not be edited, please add a warning `// Autogenerated - DO NOT MODIFY`.

Reply:
Added warning message in the README.md file.

Reply:
Otherwise we would have to add this message in every new migration.